en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md

```diff
@@ -93,17 +96,20 @@ Configure the Gateway and Publisher profiles as explained below to enable artifa
      |event_waiting_time| You can use this  element to specify an event waiting time in milliseconds (ms) for Gateway. The Gateway will wait for the time specified in this configuration after receiving an event. This ensures that artifact retrieval is not ahead of artifact updates in the Publisher; thereby, avoiding inconsistencies. The default waiting time will be 1 ms.|
-    
- 3. If the Gateway profile is running with a port offset, add the following configuration.
-     
-     ```
-     [apim.event_hub]
-     enable = true
-     service_url = "https://traffic-manager:9443"  
-     ```
 
-     - If you are not running the Gateway as an all-in-one pack and it runs with a port offset, or when it's in a different
-  node, you have to define the `event_hub service_url` element as well.
+
+#### Retrieve artifacts from the storage
+
+Once the Inbuilt Artifact Synchronizer is enabled, runtime artifacts will no longer be saved to the file system (`<API-M_HOME>/repository/deployment/server/synapse-configs/default` directory as XMLs). Instead they will be saved as blobs in the database.
+However, for debugging purposes or recovery purposes, you can use the [Gateway REST API]({{base_path}}/develop/product-apis/gateway-apis/gateway-v1/gateway-v1/#tag/Get-API-Artifacts) to view artifacts, redeploy artifacts or undeploy artifacts. 
+
+!!! note
+    Please note that Gateway REST API operations are local to that gateway deployment. If there are multiple gateway nodes in the cluster,
+    undeploying the artifacts with the REST API resource in one gateway node will not undeploy from the entire cluster. 
+
+!!! tip
+    Even though the artifacts are not getting saved to the file system, if we add a valid xml with the relevent run time artifacts(retrieved from the [Gateway REST API](https://apim.docs.wso2.com/en/3.2.0/develop/product-apis/gateway-apis/gateway-v1/gateway-v1/#tag/Get-API-Artifacts)) to the  `<API-M_HOME>/repository/deployment/server/synapse-configs/default/api` directory,
+    it will undeploy the current Artifact and deploy the API from the file system. This can be used to debug the behavior of the artifact. However this is not recommended as it can cause inconsistencies in the runtime artifacts.
```

Comment:
```suggestion
    it will undeploy the current artifact and deploy the API from the file system. This can be used to debug the behavior of the artifact. However this is not recommended as it can cause inconsistencies in the runtime artifacts.
```

Reply:
Fixed with 1269526